pkg/rancher-desktop/components/Preferences/ApplicationBehavior.vue

```diff
@@ -94,5 +72,11 @@ export default Vue.extend({
     display: flex;
     flex-direction: column;
     gap: 1rem;
+
+    .checkbox-group {
+      display: flex;
+      flex-direction: column;
+      gap: 2px;
```

Comment:
I understand that we're trying to save space, but this seems a little tight. Can we bump to 0.5rem?

Reply:
Agree. This is size was inherited from the first prototype.